modular_sand/code/modules/jobs/job_types/_job_alt_titles.dm

```diff
@@ -127,7 +128,8 @@
 		"Biomechanical Engineer",
 		"Mechatronic Engineer",
 		"Mechanic",
-		"Robotics Operator"
+		"Robotics Operator",
+		"MODsuit Engineer" // Ported from SPLURT
```

Comment:
Doesn't warrant a need for "ported from" move it to the port section of the pr description.
```suggestion
		"MODsuit Engineer"
```